Sources/SwiftDocC/Indexing/Navigator/NavigatorIndex.swift

```diff
@@ -701,8 +742,20 @@ extension NavigatorIndex {
                 multiCuratedUnvisited.remove(normalizedIdentifier)
             }
             
+            // Bump the nodes counter.
+            counter += 1
+            
+            // We only want to check for an objective-c variant
+            // if we're currently indexing a swift variant.
+            guard language == .swift else {
```

Comment:
Is this guard needed? In what cases do we have render nodes that have Objective-C variants that we don't want to index?

Reply:
It is... I'd definitely be open to suggestions for how to improve this but otherwise we end up in an infinite loop since an objective-c symbol has an objective-c variant.